ui/Tabs.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { Children, Fragment, PropsWithChildren } from "react";
+import { Tab } from "@headlessui/react";
+import { ChipButton } from "@/ui";
+
+interface TabsProps extends PropsWithChildren {
+  [key: string]: any;
```

Comment:
This is a "no no". We can't have any types, this subverts the purpose of dealing with types.

Reply:
how can we handle props we don't know what they are?

Reply:
We can't. We can handle HTML props by using HTML / React Element types. Extra props need to be defined.

Reply:
We can use generics, too
https://www.typescriptlang.org/docs/handbook/2/generics.html
https://prismic.io/blog/typescript-generics

Reply:
they actually have a generic type, `TabGroupProps`, will try that